### ISSUE TITLE
IPR039033 CAMTA1 and CAMTA2 not dbTFs

### ISSUE THREAD

**Issue by RLovering** [geneontology/go-annotation#3114]
Hi

I would appreciate it if you would revisit the IPR039033 record. Although there is some evidence that these proteins bind DNA (see #3111 where I have tried to summarise the data in papers describing these) I do not think the papers included in the IPR039033 record support:
DNA-binding transcription factor activity, RNA polymerase II-specific (GO:0000981). 

Instead more helpful mappings to GO would be 
GO:0006357 regulation of transcription by RNA polymerase II
GO:0003677 DNA binding

A Review of papers supporting this record suggests the data does not support these proteins being able to provide the genomic address because the sequence is not sufficiently unique. Although there :
PMID: 27402838 states that: "Camtas can function as co-activators with NK2 proteins" and do not provide evidence to confirm these as dbTFs.
PMID: 25049392 does provide evidence that these bind DNA sequence PyGCANTGCG, however the oligo's used include AT rich sequences that when mutated reduce the DNA binding. Thus the data does not rule out these binding AT sequences that are not sufficiently specific to provide a genomic address without the presence of a dbTF such as NK2 proteins.
PMID: 27194798 does not provide evidence of DNA binding.
PMID: 17689537 states "All CAMTAs share a similar domain organization, with a novel type of sequence-specific DNA-binding domain (designated CG-1). This domain could bind DNA directly and activate transcription, or interact with other transcription factors, not through DNA binding, thus acting as a co-activator of transcription." However, the plant and fly proteins are binding a 4-base core sequence CG(C/T)G. This is not specific enough to provide a genomic address. Plus they acknowledge that in humans DNA binding has not been shown for these proteins. Thus the CG(C/T)G motif is not with the motif PyGCANTGCG, identified above in PMID: 25049392. 
PMID: 11925432 describes the domain as reported to bind "to DNA containing thesequence motif CGCG (35)" The regions shown to bind DNA in this paper contains  the  CG-1  and  TIG  domains, but this was with genomic DNA and not a specific sequence.

Also it is not clear to me why only CAMTA1 but not CAMTA2 has these GO terms applied?
| IPRO | A2A891 | Camta1 |   | GO:0000981 | DNA-binding transcription factor activity, RNA polymerase II-specific | ECO:0000256(IEA) | ECO:0000256 | (IEA) |   | GO_REF:0000002 |   | InterPro:IPR039033

IPRO | Q9Y6Y1 | CAMTA1 |   | GO:0000981 | DNA-binding transcription factor activity, RNA polymerase II-specific | ECO:0000256(IEA) | ECO:0000256 | (IEA) |   | GO_REF:0000002 |   | InterPro:IPR039033

 Thanks

Ruth

**Comment by EBI-Hsinyu** [comment]
Hello Ruth, 
Thank you for the information. We have started the release procedure for InterPro80.0 a couple of days ago. This means that I can't edit the database for a few days and all the changes I make will  go to InterPro81.0. I will get back to you, when I can make these changes again. 
Regards, 
Hsin-Yu

**Comment by EBI-Hsinyu** [comment]
Hi Ruth, 
Thank you for this. I have changed the GO term in IPR039033 as you suggested. As for CAMTA2, we haven't integrated the signature for it yet. The signature we can use is from PANTHER subfamily. The PANTHER subfamily signatures are usually quite unstable, so we have stopped integrating them for a while now. However, we do sometimes integrate them if there is no other good signature for certain protein family. I have created a new entry IPR043661 for CAMTA2 and added the GO terms (GO:0006357 and GO:0003677). This will be visible in InterPro81.0.
Regards, 
Hsin-Yu
